Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift

```diff
@@ -37,6 +37,7 @@ class AssignmentListViewController: UIViewController, ColoredNavViewProtocol, Er
     var courseID: String!
     var env: AppEnvironment!
     var shouldFilter = false
+    var selectedIndexPath: IndexPath?
```

Comment:
`UITableView` tracks this for us. `tableView.indexPathForSelectedRow`

Reply:
thanks, i didn't know about this.